From 89bf692c08f6b2575ba677a7e65a1d603d9e4467 Mon Sep 17 00:00:00 2001
From: Dan Williams <dcbw@redhat.com>
Date: Mon, 14 May 2012 19:50:17 +0000
Subject: dbus: fix Modem interface permissions

Doesn't really matter since everything to NM is allowed in the
at_console block, but just for correctness fix it.
---
diff --git a/src/org.freedesktop.NetworkManager.conf b/src/org.freedesktop.NetworkManager.conf
index 15b11b7..501b68b 100644
--- a/src/org.freedesktop.NetworkManager.conf
+++ b/src/org.freedesktop.NetworkManager.conf
@@ -43,15 +43,12 @@
                        send_interface="org.freedesktop.NetworkManager.Connection.Active"/>
 
                 <allow send_destination="org.freedesktop.NetworkManager"
-                       send_interface="org.freedesktop.NetworkManager.Device.Cdma"/>
+                       send_interface="org.freedesktop.NetworkManager.Device.Modem"/>
 
                 <allow send_destination="org.freedesktop.NetworkManager"
                        send_interface="org.freedesktop.NetworkManager.Device.Wired"/>
 
                 <allow send_destination="org.freedesktop.NetworkManager"
-                       send_interface="org.freedesktop.NetworkManager.Device.Gsm"/>
-
-                <allow send_destination="org.freedesktop.NetworkManager"
                        send_interface="org.freedesktop.NetworkManager.Device.Serial"/>
 
                 <allow send_destination="org.freedesktop.NetworkManager"
--
cgit v0.9.0.2-2-gbebe
